src/pages/CustomerRequisitionPage.js

```diff
@@ -259,7 +267,7 @@ export const CustomerRequisition = ({
           <SearchBar
             onChangeText={onFilterData}
             value={searchTerm}
-            placeholder={`${generalStrings.search_by} ${generalStrings.item_name} ${generalStrings.or} ${generalStrings.item_code}`}
+            placeholder={`${generalStrings.search_by} ${placeholderStrings.name} ${generalStrings.or} ${placeholderStrings.code}`}
```

Comment:
could get fancy with some tagging - poor example, and probably not worth it but fun!

```
const usingIndicators = true;
const getPlaceholder = (strings, v1, v2) =>
  `${v1 ? "indicator" : "Not indicator"}${strings[0]}${
    v2 ? "indicator" : "Not indicator"
  }`;

console.log(getPlaceholder`searchBy ${usingIndicators}${usingIndicators}`);
```

Reply:
> could get fancy with some tagging - poor example, and probably not worth it but fun!
> 
> ```
> const usingIndicators = true;
> const getPlaceholder = (strings, v1, v2) =>
>   `${v1 ? "indicator" : "Not indicator"}${strings[0]}${
>     v2 ? "indicator" : "Not indicator"
>   }`;
> 
> console.log(getPlaceholder`searchBy ${usingIndicators}${usingIndicators}`);
> ```

Does look fun :smiley:. Will put it on the post-CR "nice-to-haves" list!